.github/workflows/dartx.yml

```diff
@@ -3,34 +3,47 @@ name: Dart CI
 on: [push]
 
 jobs:
+  test-windows:
+    runs-on: windows-latest
+
+    steps:
+      - name: Install dart
+        run: |
+          choco install dart-sdk
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        run: C:\tools\dart-sdk\bin\pub.bat get
+      - name: Run tests
+        run: C:\tools\dart-sdk\bin\pub.bat run test
+
   test:
     runs-on: ubuntu-latest
-    
+
     container:
       image:  google/dart:dev
-    
+
     steps:
-    - uses: actions/checkout@v1
-    - name: Install dependencies
-      run: pub get
-    - name: Run tests
-      run: pub run test
-      
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        run: pub get
+      - name: Run tests
+        run: pub run test
+
   coverage:
     runs-on: ubuntu-latest
-    
+
     container:
-      image:  google/dart:dev
-    
+      image:  google/dart:2.7
```

Comment:
Maybe we should use `stable` instead of `2.7.0`. Or even a matrix with the current stable and dev version.

Reply:
I'll change it to matrix in a separate PR where I'll also raise the minSDK to `2.7.0`